src/services/cron/reminder.cron.ts

```diff
@@ -0,0 +1,61 @@
+import cron from 'node-cron'
+import { dataSource } from '../../configs/dbConfig'
+import { MonthlyCheckInReminderService } from '../admin/reminder.service'
+import { MonthlyReminder } from '../../entities/reminder.entity'
+import Mentee from '../../entities/mentee.entity'
+import CheckIn from '../../entities/checkin.entity'
+
+export class ReminderCronService {
+  private readonly reminderService: MonthlyCheckInReminderService
+  private cronJob!: cron.ScheduledTask
+  private readonly cronSchedule: string
+
+  constructor() {
+    this.reminderService = new MonthlyCheckInReminderService(
+      dataSource.getRepository(MonthlyReminder),
+      dataSource.getRepository(Mentee),
+      dataSource.getRepository(CheckIn)
+    )
+    this.cronSchedule = process.env.REMINDER_CRON_SCHEDULE ?? '0 1 * * *' // Default to 1 AM every day
```

Comment:
```suggestion
    this.cronSchedule = process.env.REMINDER_CRON_SCHEDULE
```
if we don't have the env value let's disable the cron job